src/conduit/controllers/profile.cljs

```diff
@@ -14,6 +14,24 @@
 (defmethod control :load [_ _ state]
   {:state state})
 
+(defmethod control :load-profile [_ [id token] state]
+  {:state (assoc state :loadings? true)
+   :http (into {:endpoint :profile
+                :slug     id
+                :on-load  :load-profile-ready
+                :on-error :load-profile-error}
+               (when token {:token token}))})
```

Comment:
no need for this check here, just add `:token`, if it's nil it means there's no token anyway